app/src/main/java/org/mozilla/focus/utils/Settings.kt

```diff
@@ -388,7 +388,7 @@ class Settings(
         }
 
     var shouldUseNimbusPreview: Boolean
-        get() = preferences.getBoolean(getPreferenceKey(R.string.pref_key_use_nimbus_preview), true)
+        get() = preferences.getBoolean(getPreferenceKey(R.string.pref_key_use_nimbus_preview), false)
```

Comment:
I see [here](https://github.com/mozilla-mobile/focus-android/blob/429e032f6dadaf3a55b2ed1b629c6c6ea8bc70b7/app/src/main/res/xml/secret_settings.xml#L7) that the setting (in app settings) was set to have `false` as default but before actually changing this value to persist it to the preferences file we'd read it from here as `true` and as such initialize Nimbus with other server settings.
Nice sleuthing! 👍